src/replicache.test.ts

```diff
@@ -2099,11 +2099,13 @@ async function licenseKeyCheckTest(tc: LicenseKeyCheckTestCase) {
     fetchMock.postOnce(statusUrlMatcher, tc.mockFetchParams);
   }
 
-  const options = tc.disableLicensing
-    ? ({
+  const {enableLicensing = true} = tc;
+
+  const options = enableLicensing
+    ? {
         licenseKey: tc.licenseKey,
         disableLicensing: true,
```

Comment:
I'm unclear on why we would have an enableLicensing flag that sets a disableLicensing flag? I don't have to be clear on it tho, and feel bad for asking more questions about a boolean :)

Reply:
because its a bug :)

Reply:
it is very very weird that typescript didnt catch this...

Reply:
also odd was that the test didn't fail.  this was because of a subtle bug in the existing test, because of a gotcha with fetchMock.  fetchMock only records call it was instructed to expect.  

So, the following always passes when expectFetchCalled is false, whether or not statusUrlMatcher is fetched.
```
  if (tc.expectFetchCalled) {
    fetchMock.postOnce(statusUrlMatcher, tc.mockFetchParams);
  }
  expect(fetchMock.called(statusUrlMatcher)).to.equal(tc.expectFetchCalled); `
```

To fix this I added a 
`fetchMock.catch();`
(which is like a default expect).